tests/test_idpi/test_grib_decoder.py

```diff
@@ -0,0 +1,27 @@
+# Standard library
+from unittest.mock import Mock
+from unittest.mock import patch
+
+# Third-party
+import pytest
+
+# First-party
+from idpi.grib_decoder import GribReader
+
+earthkit = Mock()
```

Comment:
Is this necessary?

Reply:
no, good catch. It is a leftover from a previous attempt